UnityProject/Assets/Scripts/Player/LayDown.cs

```diff
@@ -0,0 +1,108 @@
+﻿using HealthV2;
+using Mirror;
+using Player.Movement;
+using UnityEngine;
+
+namespace Player
+{
+	public class LayDown : NetworkBehaviour
+	{
+		[SerializeField] private Transform sprites;
+		[SerializeField] private LivingHealthMasterBase health;
+		[SerializeField] private Rotatable playerDirectional;
+		[SerializeField] private PlayerScript playerScript;
+		[SerializeField] private Util.NetworkedLeanTween networkedLean;
+		private readonly Quaternion layingDownRotation = Quaternion.Euler(0, 0, -90);
+		private readonly Quaternion standingUp = Quaternion.Euler(0, 0, 0);
+
+		[SyncVar(hook = nameof(OnLayDown))] public bool IsLayingDown = false;
+
+
+		private void Awake()
+		{
+			playerScript ??= GetComponent<PlayerScript>();
+			playerDirectional ??= GetComponent<Rotatable>();
+			health ??= GetComponent<LivingHealthMasterBase>();
+		}
+
+		private void OnEnable()
+		{
+			if (CustomNetworkManager.IsServer == false)
+			{
+				UpdateManager.Add(EnsureCorrectState, 5f);
+			}
+		}
+
+		private void OnDisable()
+		{
+			if (CustomNetworkManager.IsServer == false)
+			{
+				UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, EnsureCorrectState);
+			}
+		}
+
+		public void EnsureCorrectState()
```

Comment:
safe guard, ensures the correct state every 5 seconds

Reply:
should be confident enough in code To not need a safeguard

Reply:
True, but until I curbstomb every mistake we make in code, this will do.
Also, it's not an uncommon thing for things to have safeguards, check any triple A game in the past 15 years, and you'll see the most common safeguard that is used everywhere; The anti-falling into the void safeguard.